### ISSUE TITLE
Amendment proposal: Definitions

### ISSUE THREAD

**Issue by adambirtles** [HackSoc/constitution#9]
§1 of the Constitution contains definitions for terms used in the rest of the document. It only defines “Society” and “Committee”.

This feels insufficient. Various other title-cased terms are used throughout the Constitution without definitions anywhere. These should probably be defined.

Conversely, “Committee” should probably not be defined here since it’s effectively define in §6.

**Comment by sersorrel** [comment]
other title-cased terms I can see:

- Societies Committee
- (Society) Member
- Computer Science
- Student Opportunities Coordinator
  - not consistently capitalised (e.g. section 7.5)
- University
  - :^)
- By-Law (as in, YUSU's By-Laws)
- Society Constitution (section 6.3)
- Students' Union
- Single Transferable Vote
  - :^^^^^^^^^^^)

of those, _I_ think it is probably worthwhile changing usages of these terms:

- Societies Committee – section 14.2 uses "the Societies Committee", the article should be removed for consistency with the rest of the document
- (Society) Member – I think this should not be title-cased (it's only currently title-cased in section 3.1, and is written "Society member" everywhere else)
- Student Opportunities Coordinator – we should consistently capitalise this
- By-Law – we should, generally speaking, probably use "YUSU By-Law"
- Society Constitution – prefer "this Constitution"
- Students' Union – prefer "YUSU"

I don't particularly think it's worth stating in the definitions section that specific terms (like "Student Opportunities Coordinator") refer to the YUSU role/document/concept; the constitution should be read and understood within the context that HackSoc is a YUSU society (possibly it's worth briefly mentioning _that_, idk; "The Society will be considered an 'Academic and Educational' YUSU Society, as defined by YUSU By-Law 10 [or maybe: by By-Law 10 of the University of York Students' Union (YUSU) By-Laws], and be represented..."?)

it occurs to me that what I'm describing is turning into a "cleanup" amendment more than a "clarify the definitions" amendment. if you agree, I would suggest that if we can gather together as many unobjectionable changes as possible into the same amendment (i.e. changes that have no practical effect on the meaning of the constitution, and certainly no changes to the intent behind it), that might a) minimise pain of voting on many tiny amendments at the AGM and b) make it easier to deal with the inevitable merge conflicts, or at least reduce the number of individual conflicts that have to be resolved (the amendment can always be presented as a few smaller commits, if that helps to describe the actual changes)

**Comment by adambirtles** [comment]
> I don't particularly think it's worth stating in the definitions section that specific terms (like "Student Opportunities Coordinator") refer to the YUSU role/document/concept

I agree. My thought for these terms was to just define YUSU and then precede their usage in the document with "YUSU('s)"

> it occurs to me that what I'm describing is turning into a "cleanup" amendment more than a "clarify the definitions" amendment. if you agree...

I do agree (see https://github.com/HackSoc/constitution/issues/10#issuecomment-784544101).
